index.html

```diff
@@ -330,7 +440,7 @@ <h3>
             section "Generating entropic identifiers for DID documents" for more
             default method and alternatives).
           </li>
-          <li>              
+          <li>
             publishing the proofed DID Document as the file
             <code>did.json</code> at the domain (qualified or unqualifed) that
```

Comment:
actual did:web backwards compatibility would mean published his DID doc at `/.well-known/` instead of or in addition to at the root, btw

Reply:
if you want to support BOTH the design goal of backwards compat AND the design goal of accomodating users without privileges to create files or folders beginning with `.`, it *might* make sense to make backwards compat an optional feature/extension, although you then have the horribly inelegant flow of creating three copies of the same signed DID doc at each update :/

Reply:
Ah, so that's what's so interesting -- that's not the case! _all_ `did:webhash` URLs will have a path fragment, so none will be top-level (none will use `.well-known` folder).